src/plugins/dashboard/public/dashboard_container/embeddable/dashboard_container.tsx

```diff
@@ -600,6 +600,15 @@ export class DashboardContainer extends Container<InheritedChildInput, Dashboard
   };
 
   public setExpandedPanelId = (newId?: string) => {
+    const dashboardWrapper = document.getElementById('dshAppWrapper');
+    if (dashboardWrapper) {
+      if (newId) {
+        dashboardWrapper.classList.add('dshAppWrapper__expandedPanel');
+      } else {
+        dashboardWrapper.classList.remove('dshAppWrapper__expandedPanel');
+      }
+    }
```

Comment:
To understand why this is necessary, you first need to know that the height of an expanded panel is determined throw flex using the `dshAppWrapper`, which has a calculated height of `100vh - <the height of any global Kibana headers>`. When a panel is expanded, it takes up **any remaining space** that the `dshAppWrapper` DIV has, and no more, which is why we (theoretically) don't get a scrollbar when expanding a panel. 

Before changing the top nav bar to sticky, the `.dashboardViewport` DIV actually extended **past** the `.dshAppWrapper` DIV - that way, when a panel was expanded, it could still respect the height of the `.dshAppWrapper`. Notice in the attached video however that we did not account for any additional banners in this, so a scrollbar was present when one or more of these banners were present - not great.